clang/lib/Driver/ToolChains/CommonArgs.cpp

```diff
@@ -1357,6 +1357,8 @@ collectSanitizerRuntimes(const ToolChain &TC, const ArgList &Args,
       if (!Args.hasArg(options::OPT_shared))
         HelperStaticRuntimes.push_back("hwasan-preinit");
     }
+    if (SanArgs.needsRadsanRt() && SanArgs.linkRuntimes())
```

Comment:
Is `SanArgs.linkRuntimes()` needed here? It doesn't fit the pattern of the other sanitizers and I wonder whether this code was updated upstream and this inconsistency is the result of a git merge?